c_tests/tests/constexpr.c

```diff
@@ -0,0 +1,31 @@
+// Compiler:
+// Run-time:
```

Comment:
Did we want to check for the presence of a constant expr somewhere in the module?

Reply:
Fixed in 5ec3b32.